[4.6.0] Cannot navigate subfolders in Java Admin Console

### What is the problem
> I get this error:
```
java.lang.BootstrapMethodError: java.lang.IllegalAccessError: no such constructor: org.exist.util.crypto.digest.MD2StreamableDigest.<init>()void/newInvokeSpecial
	at org.exist.util.crypto.digest.DigestType.<clinit>(DigestType.java:31)
	at org.exist.xmldb.RemoteCollection.getResource(RemoteCollection.java:432)
	at org.exist.client.InteractiveClient.getResources(InteractiveClient.java:409)
	at org.exist.client.InteractiveClient.process(InteractiveClient.java:575)
	at org.exist.client.ClientFrame$ProcessRunnable.run(ClientFrame.java:1443)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalAccessError: no such constructor: org.exist.util.crypto.digest.MD2StreamableDigest.<init>()void/newInvokeSpecial
	at java.lang.invoke.MethodHandleNatives.linkMethodHandleConstant(MethodHandleNatives.java:483)
	... 6 more
Caused by: java.lang.NoClassDefFoundError: org/bouncycastle/crypto/ExtendedDigest
	at java.lang.invoke.MethodHandleNatives.resolve(Native Method)
	at java.lang.invoke.MemberName$Factory.resolve(MemberName.java:975)
	at java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1000)
	at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:1394)
	at java.lang.invoke.MethodHandles$Lookup.linkMethodHandleConstant(MethodHandles.java:1750)
	at java.lang.invoke.MethodHandleNatives.linkMethodHandleConstant(MethodHandleNatives.java:477)
	... 6 more
Caused by: java.lang.ClassNotFoundException: org.bouncycastle.crypto.ExtendedDigest
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at com.sun.jnlp.JNLPClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at com.sun.jnlp.JNLPClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 12 more

```

### What did you expect
> I expect to see the content of the folder I'm double-clicking.

### Describe how to reproduce or add a test
> Open the Java Admin Console, double-click the 'apps' folder, then double-click one of your apps folders. `cd 'my-app'` results in the same error.

### Context information
Please always add the following information
- eXist-db: 4.6.0
- Java version: JDK 11.0.2
- Operating system: CentOs 7 (where eXist-db runs); OSX 13 (where Java Admin Console runs)
- 64 bit
- How is eXist-db installed? Clone from GitHub

--- a/exist-core/src/main/java/org/exist/webstart/JnlpJarFiles.java
+++ b/exist-core/src/main/java/org/exist/webstart/JnlpJarFiles.java
@@ -58,7 +58,7 @@ public class JnlpJarFiles {
             "commons-logging-%latest%",
             "commons-pool-%latest%",
             "jargo-%latest%",
-            "bcprov-jdk15on -%latest%",
+            "bcprov-jdk15on-%latest%",
             "j8fu-%latest%",
             "jackson-core-%latest%",
             "jcip-annotations-%latest%",

